packages/devtools_app/lib/src/screens/profiler/profiler_screen.dart

```diff
@@ -88,6 +85,8 @@ class _ProfilerScreenBodyState extends State<ProfilerScreenBody>
 
     cancelListeners();
 
+    addAutoDisposeListener(controller.loadingOfflineData);
```

Comment:
Why are we registering a listener with no handler?

Reply:
by default the handler is a call to setState